src/modules/index.js

```diff
@@ -0,0 +1,149 @@
+const containerElement = document.querySelector('.container-todo');
+const ulElement = document.querySelector('.ul-element');
+
+const Tasks = class {
+  constructor(description, completed = false, index = 2) {
```

Comment:
- I think it is better to not overload the function parameters with useless parameters. You can remove the `index = 2`  parameter, to keep the function simple.